### PR TITLE
Add Semantic type info.

### PR DESCRIPTION
**Pull request by htroisi**
Signed-off-by: Hannah Troisi <htroisi@pixielabs.ai>

### CODE REVIEW

File: content/en/05-reference/pxl/index.md [hunk 1]
```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
 
 ## Data Types
 
-PxL has a rich type system consisting of both concrete and semantic types.
+PxL has a rich type system consisting of both concrete and semantic types. Data types are used for the following purposes:
+
+- The query planner uses types to determine which PEM agents to pull data from.
```
Comment (philkuz) [inline content/en/05-reference/pxl/index.md:20]: this is about Semantic types, we call "concrete types" "data types" in other places. Should just call it "Semantic types"

File: content/en/05-reference/pxl/index.md [hunk 2]
```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
...
-| Type     | Description                                                        |
-| -------- | ------------------------------------------------------------------ |
-| INT64    | 64-bit integer                                                     |
-| FLOAT64  | Double precision floating point                                    |
-| TIME64NS | Time represented as 64-bit integer in nanoseconds since UNIX epoch |
-| STRING   | A UTF-8 encoded string value.                                      |
-| JSON     | A JSON value represented as a string.                              |
+The Pixie execution engine supports most concrete data types, including boolean, int64, uint128, float64, string, etc.
+
+For more info, see the complete [list of concrete data types](https://github.com/pixie-io/pixie/blob/c08aaa2c53ce95ee40817acae3f662a95994f6fb/src/api/proto/vizierpb/vizierapi.proto#L29).
```
Comment (philkuz) [inline content/en/05-reference/pxl/index.md]: I would keep the concrete types around but also reomve JSON; we don't show that.

File: content/en/05-reference/pxl/index.md [hunk 3]
```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
...
 
 ### Semantic Types
 
-This is a comprehensive list of all Kubernetes, infrastructure and metric types. For example: pod, deployment, latency_ms, etc.
+The Pixie execution engine supports many semantic data types, including those related to the following:
+
+- Kubernetes (e.g. pod name, node name, namespace name, pod status, )
```
Comment (philkuz) [inline content/en/05-reference/pxl/index.md]: trailing comma

File: content/en/05-reference/pxl/index.md [hunk 4]
```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
...
+- Infrastructure (e.g. IP address, port name, upid)
+- Metrics (e.g. bytes, duration in nanoseconds, throughput per nanosecond)
 
-More details coming soon...
+For more info, see the complete [list of semantic data types](https://github.com/pixie-io/pixie/blob/c08aaa2c53ce95ee40817acae3f662a95994f6fb/src/api/proto/vizierpb/vizierapi.proto#L51).
```
Comment (philkuz) [inline content/en/05-reference/pxl/index.md]: NIT: ~For more info, s~ See the complete

**Review by philkuz — COMMENTED** [review]
Nit: :s/Semantics/Semantic/g

**Review by philkuz — APPROVED** [review]

**Review by aimichelle — APPROVED** [review]

**Pull request merged**